lambda/MAS/Services/MailService.cs

```diff
@@ -14,6 +15,7 @@ namespace MAS.Services
     public interface IMailService
     {
         Task<Campaign> CreateAndSendDailyAsync(DateTime date, string previewText, string body, List<string> specialitiesInEmail, IEnumerable<Interest> allSpecialities, string receiveEverythingGroupId);
+        Task<Campaign> CreateAndSendWeeklyAsync(string previewText, string body, string date);
```

Comment:
The third paremter needs renaming here, as well as the implementation
```suggestion
        Task<Campaign> CreateAndSendWeeklyAsync(string previewText, string body, string title);
```